src/sqlfluff/dialects/dialect_clickhouse.py

```diff
@@ -589,6 +589,224 @@ class CreateMaterializedViewStatementSegment(BaseSegment):
     )
 
 
+class DropTableStatementSegment(ansi.DropTableStatementSegment):
+    """A `DROP TABLE` statement.
+
+    As specified in
+    https://clickhouse.com/docs/en/sql-reference/statements/drop/
+    """
+
+    type = "drop_table_statement"
+
+    match_grammar = Sequence(
+        "DROP",
+        Ref.keyword("TEMPORARY", optional=True),
+        "TABLE",
+        Ref("IfExistsGrammar", optional=True),
+        Ref("TableReferenceSegment"),
+        # Ref("OnClusterClauseSegment", optional=True),
```

Comment:
What are all of these commented out OnClusterClauseSegments for?

Reply:
It is to mind to change the ON CLUSTER block with a dedicated function 
I have 3 PRs that will refer to the future function 

do you prefer to remove it before merging? and fix this later?

Reply:
If you're gonna follow up with this soon, I'm happy to leave it in 👍